src/lib/crypt_ops/crypto_rsa_nss.c

```diff
@@ -0,0 +1,699 @@
+/* Copyright (c) 2001, Matej Pfajfar.
+ * Copyright (c) 2001-2004, Roger Dingledine.
+ * Copyright (c) 2004-2006, Roger Dingledine, Nick Mathewson.
+ * Copyright (c) 2007-2018, The Tor Project, Inc. */
+/* See LICENSE for licensing information */
+
+/**
```

Comment:
Generic comment: I'd like a log message (perhaps in `crypto_nss_late_init()`) or something saying that it's compiled with libNSS and maybe even the version? Or perhaps along with this log msg: `Tor 0.3.5.0-alpha-dev (git-537faed8034ac7f2) running on Linux with Libevent 2.1.8-stable, OpenSSL 1.1.0h, Zlib 1.2.11, Liblzma N/A, and Libz
std N/A.`

Reply:
It currently does that:

Aug 29 16:11:53.148 [notice] Tor 0.3.5.0-alpha-dev (git-e4067aeceae767da) running on Linux with Libevent 2.1.8-stable, NSS 3.38, Zlib 1.2.11, Liblzma 5.2.4, and Libzstd 1.3.5.

(That's in 6a88d8f6b413efdac4b0176cfb78431be46ca9e0 in my TLS branch)

Reply:
✓